include/mata/nfa.hh

```diff
@@ -228,6 +270,7 @@ private:
     ///
     /// These states are used in the transition relation, either on the left side or on the right side.
     /// The value is always consistent with the actual number of states in the transition relation.
+    // TODO: the name of this is very bad, lets find another. What is it actually? What about tgt_domain_size, target_domain_size, domain_size.
     size_t m_num_of_states;
```

Comment:
The variable describes how many different states are in the Post in total, both as a source and target states. `domain_size` would be OK with me. It probably better explains the meaning of the variable.